templates/snap-search.html

```diff
@@ -0,0 +1,7 @@
+{% extends "_layout.html" %}
+
+{% block title %}{{ snap_title }} — Linux software in the Snap Store{% endblock %}
+
+{% block content %}
```

Comment:
Could we give the frontends slightly more to work with, with a template something like this?

```
{% block content %}
  <ul>
  {% for snap in snaps %}
    <li>
      <h3>{{ snap.title }}</h1>
      <table>
        {% for key, value in snap.items() %}
          <tr>
            <th>{{ key }}</th>
            <td>{{ value }}</td>
          </tr>
        {% endfor %}
      </table>
    </li>
  {% endfor %}

  <nav>
    {% if links.first %}<a href="{{ links.first }}">First</a>{% endif %}
    {% if links.prev %}<a href="{{ links.prev }}">Previous</a>{% endif %}
    {% if links.next %}<a href="{{ links.next }}">Next</a>{% endif %}
    {% if links.last %}<a href="{{ links.last }}">Last</a>{% endif %}
  </nav>
{% endblock %}
```